src/frontend/scss/objects/_conversation_topic_detail.scss

```diff
@@ -44,6 +44,27 @@
         padding-top: 5px;
         border-top: 1px dashed $gray-200;
       }
+
+      blockquote {
+        background: #f9f9f9;
+        border-left: 10px solid #ccc;
+        margin: 1.5em 10px;
+        padding: 0.5em 10px;
+        quotes: '\201C''\201D''\2018''\2019';
```

Comment:
This attribute is useless here as you did not use `content: open-quote` in `:before`.

Reply:
I've played around and surely didn't clean it...thank you for your feedback